<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAQDeviceFunctionExtractionWizardPage2SetParameters</class>
 <widget class="QWizardPage" name="DAQDeviceFunctionExtractionWizardPage2SetParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="parametersLabel">
     <property name="text">
      <string>Выставьте все необходимые параметры</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="channelHorizontalLayout">
     <item>
      <widget class="QLabel" name="channelLabel">
       <property name="text">
        <string>Канал</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="channelComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="gainHorizontalLayout">
     <item>
      <widget class="QLabel" name="gainLabel">
       <property name="text">
        <string>Множитель</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="gainComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="rangeHorizontalLayout">
     <item>
      <widget class="QLabel" name="rangeLabel">
       <property name="text">
        <string>Границы</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="rangeComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="filterHorizontalLayout">
     <item>
      <widget class="QLabel" name="filterLabel">
       <property name="text">
        <string>Фильтры</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="filterComboBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
